### ISSUE TITLE
Use TabIndexes appropriately

### ISSUE THREAD

**Issue by messman** [htmllint/htmllint#72]
From [IITAA, 10.3 AND 10.4](http://www.dhs.state.il.us/IITAA/IITAAWebImplementationGuidelines.html):

"Make sure that fields appear in logical order in the HTML code or use the tabindex attribute on each field to set the appropriate order.
Note: Any element with a positive tabindex will come before all elements without tabindex in the tab order. As a result, tabindex must usually be applied to all or none of the focusable elements on a page."

"Instructions should be given within field labels if possible. If instructions are too long to fit within labels, they should be provided in an instructions section before the beginning of the form. If neither of these approaches works, consider using a technique to make the text elements focusable.
Note that disabled form fields (disabled="disabled") are non-focusable; fields may be made read-only (readonly="readonly") to keep them focusable."
